data/transactions/logic/README.md

```diff
@@ -275,6 +275,9 @@ return stack matches the name of the input value.
 | `ecdsa_pk_recover v` | for (data A, recovery id B, signature C, D) recover a public key |
 | `ecdsa_pk_decompress v` | decompress pubkey A into components X, Y |
 | `vrf_verify s` | Verify the proof B of message A against pubkey C. Returns vrf output and verification flag. |
+| `bn256_add` | for (curve points A and B) return the curve point A + B |
+| `bn256_scalar_mul` | for (curve point A, scalar K) return the curve point KA |
+| `bn256_pairing` | for (points in G1 group G1s, points in G2 group G2s), return whether they are paired => {0 or 1} |
```

Comment:
We need to get these out of the spec. They are showing up now because once the logicsig version got bumped in the logic package, the specs generation thinks they belong.

I would say you should go ahead and merge this for now, and I'll figure out how to fix it when I do the final specs merge after Foundation approval.

Reply:
what is a correct way of removing them? I followed a path (looked standard one) to bump TEAL version and these appeared in generated files. I think the source of problem a requirement (in tests?) to have `vFuture.LogicSigVersion` equal to `logic.LogicVersion`

Reply:
You did the right thing.  There is no way to remove them right now, because we've never done this before. We have code in master for the next version even as we're preparing the spec for the current version.  I'll have to change the way we generate specs so that we can be explicit that we only want the specs for version 7, even though the assembler has code for version 8.